test/notesclub/searches/remove_test.exs

```diff
@@ -0,0 +1,65 @@
+defmodule Notesclub.Searches.RemoveTest do
+  use Notesclub.DataCase
+
+  alias Notesclub.Repo
+  alias Notesclub.Searches.Remove
+  alias Notesclub.Searches.Search
+  alias Notesclub.Notebooks.Notebook
+
+  import Notesclub.SearchesFixtures
+  import Notesclub.NotebooksFixtures
+
+
+  describe "delete/0" do
+    test "function returns expected response" do
+      _search = search_fixture()
+      assert {0, nil} = Remove.eliminate()
+    end
+
+    test "should only delete older records specfied in module" do
+      search_1 = search_fixture()
+      _search_2 = search_fixture()
+
+      update_inserted_at(search_1, get_expiry_date(1))
+
+      assert {1, nil} = Remove.eliminate()
+      assert nil == Repo.get(Search, search_1.id)
```

Comment:
We could also check that `search_2` exists: `assert search2 == Repo.get(Search, search_2.id)`